docs/Integrating Your HMI/SDL Core and HMI Communication/index.md

```diff
@@ -308,19 +308,13 @@ On receipt of a request message, the server must reply with a Response. The Resp
 
 ## Error Response
 
-!!! must
-
-When an RPC encounters an error, the response object must contain the `error` property instead of the `result` property.
```

Comment:
I would keep this or change it to give better context on when to use an error response struct.

Reply:
More context on when to use each response type added in https://github.com/smartdevicelink/sdl_core_guides/pull/99/commits/51aef0e1576126f552103423b93477eb3ddde080